Sources/ArcGISToolkit/Components/OfflineMapAreasView/OfflineMapAreasView.MapViewModel.swift

```diff
@@ -0,0 +1,168 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import Combine
+import Foundation
+
+public extension OfflineMapAreasView {
```

Comment:
Does it need to be a public extension? My impression is that we will only expose the `OfflineMapAreasView` to public.